front/app/modules/commercial/customizable_homepage_banner/admin/CallToActionButtonSettings.tsx

```diff
@@ -78,15 +101,11 @@ const CallToActionButtonSettings = ({
           content={<FormattedMessage {...messages.callToActionHeader} />}
         /> */}
       </SubSectionTitle>
-      <CallToActionSettingsSigninState
-        options={CALL_TO_ACTION_NOT_SIGNED_IN_OPTIONS}
+      <CTASettingsSignedOut
         currentValue={call_to_action_not_signed_in_selected_option}
```

Comment:
You still need to work on this but: JS doesn't use snake_case for variable names. camelCase is used normally.

Reply:
But this value of `call_to_action_not_signed_in_selected_option` comes from the server as field of `app_configuration`.

And here it's convenient to use the same name for variable because I can use object desctructuring

```js
  const {
    call_to_action_not_signed_in_selected_option,
    call_to_action_signed_in_selected_option,
  } = customizable_homepage_banner;
 ```
 Otherwise, I would need to do
 ```js
   const callToActionNotSignedInSelectedOption = customizable_homepage_banner.call_to_action_not_signed_in_selected_option;
   const callToActionSignedInSelectedOption = customizable_homepage_banner.call_to_action_signed_in_selected_option;
```
Do you think it looks better?

Reply:
I think you probably won't need to because you'll need a variable, not the setting name. But to answer: the second option is more common, or you could rename after destructuring:

```
  const {
    call_to_action_not_signed_in_selected_option: callToActionNotSignedInSelectedOption ,
    call_to_action_signed_in_selected_option: callToActionSignedInSelectedOption ,
  } = customizable_homepage_banner;
```

It's also going to help if you use `cta` as you proposed 😁

Reply:
Not that DRY yet, but yes, looks better :smile:  Will fix it.